civicrm/custom/ext/gov.nysenate.webintegration/CRM/NYSS/WebIntegration/Upgrader.php

```diff
@@ -49,6 +49,32 @@ public function install() {
     }
   }
 
+  /**
+   * Upgrade from version 1.0 to 1.1
+   *
+   */
+  public function upgrade_1001() {
+    $this->ctx->log->info('Applying upgrade step 1001 in CRM_NYSS_WebIntegration_Upgrader');
+    // NYSS #16799 New field to store survey field data as JSON
+    $results = \Civi\Api4\CustomField::create(FALSE)
+      ->addValue('custom_group_id', 10)
```

Comment:
let's reference the custom_group_id via the machine name rather than ID, just in case it's different:
'custom_group_id.name' => 'Website_Survey',